test/inductor/test_flex_attention.py

```diff
@@ -6297,7 +6297,7 @@ def test_local_window_bias(self, device, params):
         bias = torch.randn(
             2 * window_size + 1,
             device=device,
-            dtype=params.dtype,
+            dtype=torch.float32,
```

Comment:
@drisspg i figured i'd fix the test by making it more accurate overall rather than fuzzing with the tolerances, what do you think?

Reply:
yeah, I like it, good fix